src/main/java/org/jabref/gui/mergeentries/newmergedialog/FieldRowView.java

```diff
@@ -212,6 +212,9 @@ public void execute() {
     }
 
     public class UnmergeCommand extends SimpleCommand {
+
+        public UnmergeCommand() { }
```

Comment:
?

Reply:
Yeah, I'll remove it. I guess I added it accidentally when resolving conflicts.